src/main/java/training/subsystems/wrist/Wrist.java

```diff
@@ -0,0 +1,66 @@
+package training.subsystems.wrist;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import training.subsystems.elbow.ElbowConstants;
+import training.subsystems.roller.Roller;
+import training.subsystems.roller.RollerConstants;
+import utils.GBSubsystem;
+
+public class Wrist extends GBSubsystem {
+
+	private  static CANSparkMax motor;
+	private static Wrist instance;
+
+	private Wrist() {
+		motor = new CANSparkMax(WristConstant.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
+		motor.getPIDController().setP(WristConstant.MOTOR_ID_P);
+		motor.getPIDController().setI(WristConstant.MOTOR_ID_I);
+		motor.getPIDController().setD(WristConstant.MOTOR_ID_D);
+	}
+
+	public static void init() {
+		if (instance == null) {
+			instance = new Wrist();
+		}
+	}
+
+	public static void goToAngel(Rotation2d targetAngle) {
```

Comment:
![image](https://github.com/user-attachments/assets/f3c73eb0-144c-4c4b-ad39-c17a320d8ba7)
consider changing to angle